### PR TITLE
Add message maker script

### PR DESCRIPTION
**Pull request by NickGrantX**
## Context
We need to be able to publish messages without hand-rolling or fiddling inside Rabbit.

## What has Changed
Added script to be able to publish messages from CLI.

## How to Test
Run the script and try it.

## Links
Trello: https://trello.com/c/myENJjmo

### CODE REVIEW

**Comment by NickGrantX** [comment]
[toolbox: ability to create messages to fix issues](https://trello.com/c/myENJjmo/1207-toolbox-ability-to-create-messages-to-fix-issues)

**Comment by NickGrantX** [comment]
[toolbox: ability to create messages to fix issues](https://trello.com/c/myENJjmo/1207-toolbox-ability-to-create-messages-to-fix-issues)

File: config.py [hunk 1]
```diff
@@ -15,3 +15,5 @@ class Config:
     DB_PORT = os.getenv('DB_PORT', '6432')
     DB_NAME = os.getenv('DB_NAME', 'postgres')
     DB_USESSL = os.getenv('DB_USESSL', '')
+    CASEAPI_HOST = os.getenv('CASEAPI_HOST', 'localhost')
```
Comment (ryangrundy7) [inline config.py:18]: Need to put these into the yml files. Currently using the default of `localhost:8161`

**Review by AdeHarrison — APPROVED** [review]
LGTM after ryans changes of course

**Review by ryangrundy7 — APPROVED** [review]
It works in my dev environment, will you need to put the case api host and port in the release yaml

File: config.py (hunk 1 above)
Comment (NickGrantX) [reply config.py:18]: done

Comment (NickGrantX) [reply config.py:18]: donee

**Pull request merged**